index.html

```diff
@@ -0,0 +1,83 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>YouTube video page</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <div class="navbar">
+		<span class="menu-button"></span>
+		<img class="logo" src="./img/logo.png" />
+		<div class="navbar-right">
+		  <input class="search-input" placeholder="Search" /> 
+		  <button class="search-button">
+			<span class="search-button-icon"></span>
+		  </button> 
+		  <span class="upload-button"></span>
+		  <span class="history-button"></span>
+		  <span class="help-button"></span>
+		  <button class="signin-button">Sign In</button>
+		</div>
+      </div>
+      <div class="content">
```

Comment:
You've structured the project really well, however, please use the semantic HTML5 <section> tag for the different page sections.